Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -33,20 +35,63 @@ public GameObject ThumbRoot
             }
         }
 
-        [Range(0, 1)]
+        #region MyValue 
+        public int MinValue = -4;
+        public int MaxValue = 4;
+
+        public bool WholeNumbers = true;
+
+        public float startValue = 0f;
+
+        [Tooltip("Automatic placement of slider's steps.")]
         [SerializeField]
+        private bool TickMarksAutoFill = true;
```

Comment:
The repo [has typically used camelCase](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#naming-conventions) for private fields.